app/controllers/api/v1/application_controller.rb

```diff
@@ -0,0 +1,9 @@
+class Api::V1::ApplicationController < ActionController::Base
+  include CurrentUser
+
+  private
+
+  def warden_strategies
```

Comment:
a comment like this might help: `# overrides CurrentUser version to only allow basic auth`